### PR TITLE
Changed CONTRIBUTING.md

### LINKED ISSUE #825: Update README Template
# Intro

Recently we removed all the Images from the Repository, the images mostly lie in the READMEs, we now prefer using Image Hosting websites in order to keep the repo size in check.

# Problem

The [Readme Template](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) still hasn't been updated accordingly.

# Solution

A small GIF, explaining how an image can be hosted on a platform like `imgur`, along with generating the link and pasting it in a Markdown file.

Any other solution is also acceptable, but we will prefer a visual means so that it is easy to comprehend.

### PR DESCRIPTION
**Pull request by pi-squared-4**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #825

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

# Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,6 +95,21 @@ Then add a title and description to your pull request that explains your contrib
...
+- We recently have removed all the images and screenshots from our repository and linked them to markdown files.
+    
+    ## How to do that? ##
+
+    - You can do that by hosting all you images and screenshots to any images hosting site such as [imgur](www.imgur.com).
+    - Then link your uploaded images to readme files. An instance 👇
+    
+    ![update_readme](https://media.giphy.com/media/45Bc4gO8j4D0oCLg0g/giphy.gif)
+    
+###### (This helps us save a lot of precious space.) ######
```
Comment (vybhav72954) [inline CONTRIBUTING.md:109]: I don't think we need this line. And as a rule of thumb, we should be using the same Heading Style for every Heading used.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,6 +95,21 @@ Then add a title and description to your pull request that explains your contrib
 <img width="882" alt="pullr" src="https://user-images.githubusercontent.com/44089458/86092345-fed8c280-baca-11ea-9a04-95991d9c60d2.PNG">  
 - Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged.🥳
 
+### Note : Do not add images, rather 👇
+
+- We recently have removed all the images and screenshots from our repository and linked them to markdown files.
+    
+    ## How to do that? ##
```
Comment (vybhav72954) [inline CONTRIBUTING.md:102]: You are using an h2 after h3.

The Markdown Styling will report that as an error. Also, different types of Syntax shouldn't be used.

So either use h4 or make the line bold.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -95,6 +95,21 @@ Then add a title and description to your pull request that explains your contrib
...
 - Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged.🥳
 
+### Note : Do not add images, rather 👇
+
+- We recently have removed all the images and screenshots from our repository and linked them to markdown files.
+    
+    ## How to do that? ##
+
+    - You can do that by hosting all you images and screenshots to any images hosting site such as [imgur](www.imgur.com).
+    - Then link your uploaded images to readme files. An instance 👇
```
Comment (vybhav72954) [inline CONTRIBUTING.md:105]: Can you make this `README`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -95,6 +95,21 @@ Then add a title and description to your pull request that explains your contrib
 <img width="882" alt="pullr" src="https://user-images.githubusercontent.com/44089458/86092345-fed8c280-baca-11ea-9a04-95991d9c60d2.PNG">  
 - Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged.🥳
 
+### Note : Do not add images, rather 👇
+
+- We recently have removed all the images and screenshots from our repository and linked them to markdown files.
+    
+    ## How to do that? ##
+
+    - You can do that by hosting all you images and screenshots to any images hosting site such as [imgur](www.imgur.com).
```
Comment (vybhav72954) [inline CONTRIBUTING.md:104]: Would it be possible that you add links to imgbb and postimage as well

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Just a few minor changes here and there

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -95,20 +95,19 @@ Then add a title and description to your pull request that explains your contrib
 <img width="882" alt="pullr" src="https://user-images.githubusercontent.com/44089458/86092345-fed8c280-baca-11ea-9a04-95991d9c60d2.PNG">  
 - Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged.🥳
 
-### Note : Do not add images, rather 👇
+### Note : Do not add images, rather 👇 ###
 
 - We recently have removed all the images and screenshots from our repository and linked them to markdown files.
     
-    ## How to do that? ##
+    #### How to do that? ####
```
Comment (vybhav72954) [inline CONTRIBUTING.md:102]: Hi there, all these final changes look good apart from one tiny mistake

```
H4 you have used - #### How to do that? ####
H4 that should be used - #### How to do that?
```

I don't remember their exact names, but both the style of Heading is correct, but you should follow only 1 specific method. So please change this.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by vybhav72954** [comment]
One more thing. Please Link the issue to this PR.

In the body of your Pull Request, there is a Fixes<number> string.

Kindly change the number to the Number of the Issue.

This will look like this

```
Fixes#825
```

**Review by vybhav72954 — APPROVED** [review]
LGTM

**Comment by pi-squared-4** [comment]
Is there something wrong left?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -101,7 +101,7 @@ Then add a title and description to your pull request that explains your contrib
     
     #### How to do that? 
 
-    - You can do that by hosting all you images and screenshots to any images hosting site such as [imgur]('https://imgur.com/'), [imgbb]('https://imgbb.com/'), [postimages]('https://postimages.org/').
+    - You can do that by hosting all you images and screenshots to any images hosting sites such as [imgur]('https://imgur.com/'), [imgbb]('https://imgbb.com/'), [postimages]('https://postimages.org/').
```
Comment (vybhav72954) [inline CONTRIBUTING.md:104]: I asked you to get rid of quotes, not put quotes everywhere

```
[imgur](https://imgur.com/)
[imgbb](https://imgbb.com/)
```

This is what you should do.

Please get this done

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -99,7 +99,7 @@ Then add a title and description to your pull request that explains your contrib
```
[the remark this answers is not in the document]
Comment (vybhav72954) [reply CONTRIBUTING.md:99]: @pi-squared-4 You are yet to do this change, please note that

```
### Note : Do not add images, rather 👇 ###
```
**is wrong**.

and it should be

```
### Note : Do not add images, rather 👇
```

**This is is correct**

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
@pi-squared-4 Hey there. There are 2 changes that you need to make.

First is the trailing hashtags in the heading.

Secondly, PLease get rid of quotes, i said that because of quotes the link is not working, so rather than adding quotes everywhere, you are supposed to remove them, I have provided an example as well.

Please try to do this in one commit only.

**Comment by pi-squared-4** [comment]
I am changing it, but I dont know how everytime, it is reverted back.

**Comment by vybhav72954** [comment]
> I am changing it, but I dont know how everytime, it is reverted back.
> I have changed it already twice.

@pi-squared-4 It worked 🥳 🥳 

Now please sort out the links and I will merge this right away.

File: CONTRIBUTING.md (hunk 6 above)
Comment (vybhav72954) [reply CONTRIBUTING.md:104]: @pi-squared-4 Just this small change

**Comment by vybhav72954** [comment]
LGTM 💯

**Comment by pi-squared-4** [comment]
I cannot see any points of my gssoc profile, can you help me out?

**Comment by vybhav72954** [comment]
> I cannot see any points of my gssoc profile, can you help me out?

The leaderboard is yet to be finalised

**Pull request merged**